redis/asyncio/client.py

```diff
@@ -672,12 +672,12 @@ def __init__(
             self.encoder = self.connection_pool.get_encoder()
         if self.encoder.decode_responses:
             self.health_check_response: Iterable[Union[str, bytes]] = [
```

Comment:
IMHO you can remove the Iterable piece in 674. I know you didn't add it, but it's in function, so has no value.